mangaxyz.py

```
@@ -139,7 +139,11 @@ async def find_chapters(self, manga_id: str):
 
     async def find_chapter_pages(self, chapter_id: str):
         async with httpx.AsyncClient(timeout=TIMEOUT) as client:
-            html_page = await self.fetch(client, f"{self.api}/{chapter_id}")
+            try:
+                html_page = await self.fetch(client, f"{self.api}/{chapter_id}")
+            except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)